src/example.pl

```diff
@@ -6,3 +6,9 @@
 foreach my $ele (@array) {
     print "$ele\n";
 }
+
+my @array2 = ("Nyan", "Cat");
+
+foreach my $ele (@array2) {
+    print "$ele\n";
```

Comment:
**[perlcritic]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
**Return value of flagged function ignored - print**, near <code>print "$ele\n";</code>.<br>(Ref: [InputOutput::RequireCheckedSyscalls](https://metacpan.org/pod/Perl::Critic::Policy::InputOutput::RequireCheckedSyscalls))